src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py

```diff
@@ -85,7 +84,7 @@ def __init__(self, # validation_set, # net_params,
         out = _tf.nn.xw_plus_b(fc1, weights=weights["drawing_dense1_weight"],
             biases=biases["drawing_dense1_bias"])
         out = _tf.nn.softmax(out)
-
+        
```

Comment:
Please don't introduce trailing whitespace

Reply:
Sorry about that.